server/config/default.js

```diff
@@ -24,6 +24,12 @@ module.exports = {
   graph: {
     url: 'https://graph-qa.fuse.io/subgraphs/name/fuseio/fuse-qa'
   },
+  jobsGases: {
```

Comment:
it's the gas limit for tx than for the job. what you say to rename it to something like `gasLimitForTx` ?